culane_metric/evaluate.py

```diff
@@ -0,0 +1,205 @@
+""" Evaluation script for the CULane metric on the LLAMAS dataset.
+
+This script will compute the F1, precision and recall metrics as described in the CULane benchmark.
+
+The predictions format is the same one used in the CULane benchmark.
+In summary, for every annotation file:
+    labels/a/b/c.json
+There should be a prediction file:
+    predictions/a/b/c.lines.txt
```

Comment:
Could hint that a would be from {train | validation | test}
and b is the recording name
c is the actual frame

Reply:
```suggestion
In summary, for every annotation file:
    labels/a/b/c.json
There should be a prediction file:
    predictions/a/b/c.lines.txt
For example, given `anno_dir=valid` and `pred_dir=valid_preds` (as in the CLI arguments), for this file:
    valid/images-2014-12-22-12-35-10_mapping_280S_ramps/1419280602_0873282000.json
This one should exist:
    valid_preds/images-2014-12-22-12-35-10_mapping_280S_ramps/1419280602_0873282000.lines.txt

```
I added an example to make it more clear.